cmd/plaid.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"github.com/quesurifn/go-digit/banker"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultPublicToken string = "replaceme"
+)
+
+// Flags
+var (
+	publicToken string
+)
+
+var plaidCmd = &cobra.Command{
+	Use:   "plaid",
+	Short: "Wraps useful plaid commands",
+	Long:  "Wraps useful plaid commands",
+}
+
+var plaidExchangeTokenCmd = &cobra.Command{
```

Comment:
I added this CLI command so we could test the exchanging of the public token for an access token. 

You just call it by doing the following:

`go run main.go plaid exchange --token="tokenhere"`